CONTRIBUTING.md

```diff
@@ -0,0 +1,183 @@
+
+    `git clone git@github.com:eventbrite/britecharts-react.git`
+
+ + Change directory into the repository root and install its dependencies:
+
+    `yarn install`
+
+ + In the root or the repository folder, run:
+
+    `yarn run test`
+
+ + In a second terminal window, run:
+
+    `yarn run storybook`
+
+
+This process will generate the storybook docs and show them in ``localhost:6006``. You can use this demos as your testing platform when creating new charts.
+
+
+## Modifying a chart
+
+We have created this components with Tests First, and we encourage you to do the same. However, you can also write tests later.
+
+For a TDD workflow, the process would look like this:
+
+1. Create a new branch for your modification
+1. On a new terminal, run ``yarn start`` and ``yarn run storybook`` and navigate to the chart's demo or create a new demo in ``/src/stories/index.js``
+1. Find the test of the chart you want to modify in ``/src/charts/*/*.test.js``
+1. Write a failing test for the feature you want to add
+1. On a new terminal, run ``yarn run test:watch``
+1. Check that the test fail
+1. Write the code that would make that test pass on the chart located in ``/src/charts/*/*Component.js`` or ``/src/charts/*/*Chart.js``. Please follow the code style in the current files.
+1. Make the test pass by writing the less amount of code
+1. Refactor until you get a nice and clean code
+1. Check the demo to see the code in action
+1. Update the documentation so that the proper demos gets generated when you run ``yarn run build:storybook``
+1. Create a pull request and ask people of the team to review it
+1. Once you have the OK, merge it!
+
+
+## Creating a new chart
+
+Adding a new chart is a bunch of work, but we hope that using the current code and documentation will help in the process.
+
+1. Create a new branch for your new chart
+1. On a new terminal, run ``yarn start`` ``yarn run storybook`` to get ready the demos and turn on the dev server.
+1. Given that you new chart is ``ChartName``, create in ``/src/charts/ChartName`` a new file ``ChartName.js`` and test file ``ChartName.test.js``.
+1. Create also a new component in ``/src/charts/ChartName`` a new file ``ChartNameComponent.js`` and test file ``ChartNameComponent.test.js``.
+1. Add the new demo entry on ``/src/stories/index.js``, keeping the same format.
+7. Go to ``webpack.config.js`` and add the new chart to the ``CHARTS`` object, follow the same style.
+8. Go to your test file ``ChartName.test.js`` and, taking another test as an example, copy over pieces until you have the first test case.
+1. Add one test and make it fail callin ``yarn run test:watch``.
+1. Keep on coding according to the code style you can see in the current charts, using the stackedAreaChart as a reference.
+1. Generate your docs with ``yarn run build:storybook`` and manually test the demos.
+1. Create a pull request with your branch and ping one of the core authors to get it reviewed.
+1. Once you have a shipit, merge it
```

Comment:
Same with these numbers. Oh wait maybe Markdown will turn them into the correct ones?

Reply:
Yep, Markdown will do it!